database/util.go

```diff
@@ -6,14 +6,15 @@ import (
 	"strings"
 )
 
-const advisoryLockIdSalt uint = 1486364155
+const advisoryLockIDSalt uint = 1486364155
 
 // GenerateAdvisoryLockId inspired by rails migrations, see https://goo.gl/8o9bCT
+// nolint: golint
```

Comment:
I meant that this should be on the same line as the function declaration. e.g. `func GenerateAdvisoryLockId(databaseName string, additionalNames ...string) (string, error) {  // nolint: golint`